### ISSUE TITLE
WebApp:az webapp config ssl create results in UnknownError: Unknown error

### ISSUE THREAD

**Issue by jeromewaibel** [Azure/azure-cli#16181]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

## Describe the bug

**Command Name**
`az webapp config ssl create`

**Errors:**
```
UnknownError: Unknown error
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- Create a web app
- Create a binding with `az webapp config hostname add --hostname {} --webapp-name {} --resource-group {} --subscription {}`
- Run `az webapp config ssl create --debug --hostname {} --name {} --resource-group {} --subscription {}`

## Expected Behavior

* A certificate should be generated
* No error should happen

## Environment Summary
```
Linux-4.19.128-microsoft-standard-x86_64-with-debian-bullseye-sid
Python 3.6.10
Installer: DEB

azure-cli 2.15.1

```
## Additional Context

App Service Plan and App Service are located in different resource groups.
Though the command yields an error, a certificate is still generated in the resource group of the App Service Plan.

Details when running with --debug:

```
Last HTTP Request-URL:

msrest.http_logger : Request URL: 'https://management.azure.com/subscriptions/.../resourceGroups/.../providers/Microsoft.Web/certificates/...?api-version=2019-08-01'
msrest.http_logger : Request method: 'PUT'
msrest.http_logger : Response status: 202
msrest.http_logger : Location': 'https://management.azure.com/subscriptions/.../resourceGroups/.../providers/Microsoft.Web/certificates/.../operationresults/...?api-version=2019-08-01'
msrest.http_logger :     'Content-Length': '0'
msrest.http_logger : Response content:
msrest.http_logger :

Trace:

msrest.exceptions : Unknown error
azure.cli.core.util.handle_exception is called with an exception:
Traceback (most recent call last):
  File "/opt/az/lib/python3.6/site-packages/knack/cli.py", line 215, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 654, in execute
    raise ex
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 717, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 709, in _run_job
    return cmd_copy.exception_handler(ex)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/appservice/commands.py", line 37, in _ex_handler
    raise ex
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 688, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 325, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/__init__.py", line 784, in default_command_handler
    return op(**command_args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/appservice/custom.py", line 2456, in create_managed_ssl_cert
    certificate_envelope=easy_cert_def)
  File "/opt/az/lib/python3.6/site-packages/azure/mgmt/web/v2019_08_01/operations/_certificates_operations.py", line 300, in create_or_update
    raise models.DefaultErrorResponseException(self._deserialize, response)
azure.mgmt.web.v2019_08_01.models._models_py3.DefaultErrorResponseException: Unknown error

cli.azure.cli.core.azclierror : UnknownError: Unknown error
UnknownError: Unknown error
Event: Cli.PostExecute [<function AzCliLogging.deinit_cmd_metadata_logging at 0x7efca516cea0>]
```

**Comment by yungezz** [comment]
route to appropriate team. cc @houk-ms

**Comment by yungezz** [comment]
https://github.com/Azure/azure-cli/issues/14287

**Comment by calvinsID** [comment]
Fixed by #16124. Please update to latest CLI (2.16) for this fix
